src/LogicItLab/Salesforce/MapperBundle/TestHelper.php

```diff
@@ -0,0 +1,193 @@
+<?php
+
+namespace LogicItLab\Salesforce\MapperBundle;
+
+use LogicItLab\Salesforce\MapperBundle\Annotation\AnnotationReader;
+use LogicItLab\Salesforce\MapperBundle\Exception\FileHandlerNotInitializedException;
+use Symfony\Component\Finder\Finder;
+
+class TestHelper
```

Comment:
What do you think of a name like `wsdlValidator`? I'm not sure if `TestHelper` is very clear on what exactly it does.